frontend/models/User.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+    handle: {
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
```

Comment:
we need the following data for users:
fname,
lname, 
email,
phone,
password,


all of which are strings